pkg/devicetemplates/ttscsv.go

```diff
@@ -402,6 +465,24 @@ func (*ttsCSV) Convert(ctx context.Context, r io.Reader, ch chan<- *ttnpb.EndDev
 	}
 }
 
+type profileIdentifiersKeyType struct{}
+
+var profileIdsKey = &profileIdentifiersKeyType{}
```

Comment:
`profileIDs`; it's unavoidable with protobuf generated code that we can use `IDs`.